ee_tests/Dockerfile.builder

```diff
@@ -47,6 +47,10 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
+# install zabbix-sender
+RUN  wget https://repo.zabbix.com/zabbix/3.0/rhel/7/x86_64/zabbix-sender-3.0.9-1.el7.x86_64.rpm \
+&& yum -y --setopt tsflags='nodocs' install zabbix-sender-3.0.9-1.el7.x86_64.rpm
```

Comment:
Why not `yum -y --setopt tsflags='nodocs' install https://repo.zabbix.com/zabbix/3.0/rhel/7/x86_64/zabbix-sender-3.0.9-1.el7.x86_64.rpm`?